deltacat/compute/compactor_v2/compaction_session.py

```diff
@@ -517,47 +395,11 @@ def merge_input_provider(index, item):
 
     compaction_audit.set_records_deduped(total_dd_record_count.item())
     compaction_audit.set_records_deleted(total_deleted_record_count.item())
```

Comment:
As a rule, keep audit setting near to the data. Hence, set audit came out of merge step in the `_run_merge` method.

Reply:
Yes, audit must be updated as soon as the data is calculated.